src/pages/Compare/index.tsx

```diff
@@ -189,8 +192,12 @@ export class CompareBase extends React.Component<Props> {
         );
       }
 
-      if (versionFile && nextFilePath) {
-        if (!nextFileIsLoading && nextFile === undefined) {
+      if (versionFileIsReady && nextFilePath) {
```

Comment:
As discussed above, if you change the name this could be `if (!shouldFetchVersionFile && nextFilePath)`.